doc: add role management (#520)
harvester/docs: docs/host/host.md

![delete.png](/img/v1.2/host/delete-node.png)

> ## Roles Management
> 
> It's very common for users to need to replace the existing management node due to hardware issues. However, in previous Harvester versions, it was not easy to accurately promote a specific worker node to a management node.
> 
> To make it more flexible and easier, Harvester introduces various roles in v1.3.0. The roles are the following:

Reviewer: ```suggestion
## Role Management

Hardware issues may force you to replace the management node. In earlier Harvester versions, accurately promoting a specific worker node to a management node was not easy. Harvester v1.3.0 improves the process by introducing the following roles:
```

> - **Management**: If the node is a management role, it is highly prioritized to be promoted to the management nodes.
> - **Worker**: If the node is a worker role, it will never be promoted to the management nodes.
> - **Witness**: If the node is a witness role, it will only be an etcd node in this cluster.

Reviewer: ```suggestion
- **Management**: Allows a node to be prioritized when Harvester promotes nodes to management nodes.
- **Witness**: Restricts a node to being a witness node (only functions as an etcd node) in a specific cluster.
- **Worker**: Restricts a node to being a worker node (never promoted to management node) in a specific cluster.
```

> :::caution
> 
> Only one witness node is allowed in the cluster for now.
> 
> :::
> 
> Please check the installation guide [here](/v1.3/install/index) to see how to set the roles during the installation.

Reviewer: ```suggestion
:::caution
Harvester currently allows only one witness node in the cluster.
:::

For more information about assigning roles to nodes, see [ISO Installation](/v1.3/install/index).
```
